htdocs/core/class/commondocgenerator.class.php

```diff
@@ -397,6 +397,8 @@ public function get_substitutionarray_object($object, $outputlangs, $array_key =
 		$array_key.'_payment_term_code'=>$object->cond_reglement_code,
 		$array_key.'_payment_term'=>($outputlangs->transnoentitiesnoconv('PaymentCondition'.$object->cond_reglement_code) != 'PaymentCondition'.$object->cond_reglement_code ? $outputlangs->transnoentitiesnoconv('PaymentCondition'.$object->cond_reglement_code) : ($object->cond_reglement_doc ? $object->cond_reglement_doc : $object->cond_reglement)),
 
+        $array_key.'_incoterms'=>$object->display_incoterms(),
```

Comment:
With v13, all object has not necessary the display_incoterms() method available. Only object that use the "trait" CommonIncoterm. So to avoid error with object with method not available can you add a test
if (method_exists())

Reply:
Good point. Will implement this